util/file_reader_writer.cc

```diff
@@ -453,7 +463,13 @@ Status WritableFileWriter::WriteBuffered(const char* data, size_t size) {
         old_size = next_write_offset_;
       }
 #endif
-      s = writable_file_->Append(Slice(src, allowed));
+      {
+        auto prev_perf_level = GetPerfLevel();
+//        SetPerfLevel(PerfLevel::kEnableTime);
+        IOSTATS_CPU_TIMER_GUARD(cpu_write_nanos, env_);
```

Comment:
I think fsync() should be included too, because there most iowait CPU is spent. I think it's in function `WritableFileWriter::SyncInternal`.